src/main/java/com/compstore/dto/PagingAndSortingRequestDTO.java

```diff
@@ -0,0 +1,13 @@
+package com.compstore.dto;
+
+import lombok.Data;
+
+@Data
+public class PagingAndSortingRequestDTO {
```

Comment:
Request do żądania paginacji i sortowania. Ustawione wartości domyślne - to znaczy, że jak nikt nie poda obiektu na froncie to wyszukiwanie zajdzie z podstawowymi parametrami. Na razie w MVP sortujemy tylko po cenie więc nie przekazuję w tym obiekcie nazw parametrów sortowania